types/node/net.d.ts

```diff
@@ -37,20 +38,12 @@ declare module "net" {
         buffer: Uint8Array | (() => Uint8Array);
         /**
          * This function is called for every chunk of incoming data.
-         * Two arguments are passed to it: the number of bytes written to buffer and a reference to buffer.
-         * Return false from this function to implicitly pause() the socket.
+         * Two arguments are passed to it: the number of bytes written to `buffer` and a reference to `buffer`.
+         * Return `false` from this function to implicitly `pause()` the socket.
          */
-        callback(bytesWritten: number, buf: Uint8Array): boolean;
-    }
-    interface ConnectOpts {
```

Comment:
This was previously exported; does it need to stay for compat?

Reply:
Considered. Its only real purpose was to be extended by other `*ConnectOpts` interfaces within that module, and doesn't make much sense as a type in isolation; I reasoned that if there was a usage case for which removing it would generate errors, then exporting it with its only property removed would probably also generate errors. Happy to be guided, though.

Reply:
I did find at least one project which depends on this type: https://github.com/nxtedition/node-http2-proxy/blob/68e79d12bafba58b57b1a53114a9f0e8a0d4c164/index.d.ts#L77

Reply:
Good catch, thanks.